internal/app/go-filecoin/node/block_propagate_test.go

```diff
@@ -77,7 +78,7 @@ func TestChainSync(t *testing.T) {
 
 	ConnectNodes(t, nodes[0], nodes[1])
 
-	// Advance node's time so that it is epoch 1
+	fmt.Printf("miner starting head: %s\n", nodes[0].PorcelainAPI.ChainHeadKey())
```

Comment:
Did you mean to leave this here? It's probably best to use t.Log if you want output.